src/serac/numerics/functional/boundary_integral_kernels.hpp

```diff
@@ -386,17 +188,17 @@ auto batch_apply_chain_rule(derivative_type* qf_derivatives, const tensor<T, n>&
  * @see mfem::GeometricFactors
  * @param[in] num_elements The number of elements in the mesh
  */
-template <Geometry g, typename test, typename trial, int Q, typename derivatives_type>
-void action_of_gradient_kernel(const mfem::Vector& dU, mfem::Vector& dR,
-                               CPUArrayView<derivatives_type, 2> qf_derivatives, std::size_t num_elements)
+template <int Q, mfem::Geometry::Type geom, typename test, typename trial, typename derivatives_type>
```

Comment:
How is `Q` interpreted for non-tensor-product elements? Does the comment need to be updated?

Reply:
I'm not going to worry about this too much, since the quadrature rules are being revamped imminently (and many of these template parameters will be gone).